Makefile

```diff
@@ -1,7 +1,7 @@
 BRANCH = "master"
 REPONAME = "neo-go"
 NETMODE ?= "privnet"
-BINARY = "./bin/neo-go"
+BINARY = '$(shell if [ "$(shell go env GOOS)" = "windows" ]; then echo "./bin/neo-go.exe"; else echo "./bin/neo-go"; fi)'
```

Comment:
How about this?
```
EXTENSION=
ifeq ($(GOOS),windows)
  EXTENSION=".exe"
endif
BINARY=./bin/neo-go$(EXTENSION)
```

Reply:
`$(GOOS)` doesn't work for me neither from Makefile, nor from the windows shell, but `$(shell go env GOOS)` works fine. Maybe I have the wrong ENV settings.

Fixed.

Reply:
Yeah, it's probably not a part of regular environment, so you get it from `go`.